src/TranslationUi/Term/Repository.php

```diff
@@ -0,0 +1,45 @@
+<?php # -*- coding: utf-8 -*-
+/*
+ * This file is part of the MultilingualPress Yoast Seo Sync package.
+ *
+ * (c) Inpsyde GmbH
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Inpsyde\MultilingualPress\YoastSeoSync\TranslationUi\Term;
+
+use Inpsyde\MultilingualPress\TranslationUi\Term\RelationshipContext;
+
+class Repository
+{
+    const OPTION_KEY = 'wpseo_taxonomy_meta';
+
+    /**
+     * @param RelationshipContext $relationshipContext
+     * @param string $optionKey
+     * @return string
+     */
+    public function optionByContext(RelationshipContext $relationshipContext, string $optionKey): string
```

Comment:
If the method throw an exception would be nice to have it in the docblock so IDE can take advance of it and give right feedback to the client.